src/Text/LLVM/Triple/AST.hs

```diff
@@ -0,0 +1,467 @@
+{- |
+Module      : Text.LLVM.Triple.AST
+Description : AST of LLVM target triples.
+License     : BSD3
+Maintainer  : Langston Barrett
+Stability   : experimental
+-}
+
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE StrictData #-}
+
+module Text.LLVM.Triple.AST
```

Comment:
Do you think it is worth reexporting this module from `Text.LLVM.AST`?

Reply:
> Do you think it is worth reexporting this module from Text.LLVM.AST?

Quite frankly, I don't have any heuristics at all about when it is good to re-export things. I'm open to suggestions, and would especially appreciate any general guidance you have on the subject!

Reply:
I don't have anything resembling heuristics, only intuition. I bring this up since it's a bit odd that `Text.LLVM.AST` now exports all of the AST types _except_ the target triple–related ones, which intuitively feels a bit strange. Ultimately, it's little more than an intuition, so I'll leave it up to you to decide whether it's worth doing or not.

Reply:
I'm going to leave this off for now because `Text.LLVM.AST` lacks an explicit export list, so it would be a bit of a pain.